### PR TITLE
loops-simple_programs_with_repitition-easy-q1

### PR DESCRIPTION
**Pull request by SanaBob**
+ loops_simple_programs_with_repitition_q1

### CODE REVIEW

**Review by Hedgemon4 — CHANGES_REQUESTED** [review]
Looks good, might just want to change the repetition number to something different from the base one, like 15 or something to make sure they don't hardcode it

**Review by SamamthaH — DISMISSED** [review]

File: src/loops/simple_programs_with_repitition/easy/q1/PunishmentLoopTest.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+    public Clause[] testSentence() {
+
+        Clause[] output = new Clause[repetitions * 2 + 3];
+
+        output[0] = new StringLiteral("Enter a sentence: ");
+        output[1] = new NewLine();
+        output[2] = new StringLiteral("Enter the number of times to repeat the sentence: ");
+
+        for (int i = 0; i < repetitions; i++){
```
Comment (Opeyem1a) [inline src/loops/simple_programs_with_repitition/easy/q1/PunishmentLoopTest.java:20]: Nice

**Review by Opeyem1a — APPROVED** [review]
Looks good to me - but we'll need to modify this to take different inputs, for now make `sentence` something other than hello world so it's a tiny bit sturdier

File: src/loops/simple_programs_with_repitition/easy/q1/PunishmentLoopTest.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
+package loops.simple_programs_with_repitition.easy.q1;
+
+import global.variables.*;
+import global.variables.clauses.*;
+import global.BaseTest;
+
+public class PunishmentLoopTest extends BaseTest {
```
Comment (Opeyem1a) [inline src/loops/simple_programs_with_repitition/easy/q1/PunishmentLoopTest.java:7]: Rename to `MainTest` and comment if Parson's

**Review by Opeyem1a — CHANGES_REQUESTED** [review]

File: src/loops/simple_programs_with_repitition/easy/q1/MainTest.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
+package loops.simple_programs_with_repitition.easy.q1;
+
+import global.variables.*;
+import global.variables.clauses.*;
+import global.BaseTest;
+
+public class MainTest extends BaseTest { //Parson's Test
+
```
Comment (Hedgemon4) [inline src/loops/simple_programs_with_repitition/easy/q1/MainTest.java:8]: I think the parsons label should be on the next line

File: src/loops/simple_programs_with_repitition/easy/q1/MainTest.java [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+import global.variables.*;
+import global.variables.clauses.*;
+import global.BaseTest;
+
+public class MainTest extends BaseTest { //Parson's Test
+    
+    String sentence = "This is an example sentence.";
+    int repetitions = 10;
+
+    public Clause[] testSentence() {
```
Comment (Hedgemon4) [inline src/loops/simple_programs_with_repitition/easy/q1/MainTest.java:12]: I think this should be redone as an inputTest

**Review by Hedgemon4 — CHANGES_REQUESTED** [review]

File: src/loops/simple_programs_with_repitition/easy/q1/MainTest.java (hunk 3 above)
Comment (SamamthaH) [reply src/loops/simple_programs_with_repitition/easy/q1/MainTest.java:8]: Specifically it needs to be // Parsons

**Review by Opeyem1a — CHANGES_REQUESTED** [review]

**Comment by SanaBob** [comment]
No option for InputTest just yet (number of placeholders is unknown)

**Comment by Opeyem1a** [comment]
#117 Should fix this when resolved, switching this to a draft PR until then

**Pull request merged**